<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>教师资料管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="selectBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查询信息</string>
    </property>
    <property name="icon">
     <iconset resource="file.qrc">
      <normaloff>:/image/images/select.png</normaloff>:/image/images/select.png</iconset>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>701</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="selectPage">
     <widget class="QComboBox" name="selTableComBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>文档信息</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>教师信息</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>班级信息</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>授课信息</string>
       </property>
      </item>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>681</width>
        <height>331</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="selTeachComBox">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="maxVisibleItems">
       <number>100</number>
      </property>
     </widget>
     <widget class="QComboBox" name="selClassComBox">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="selTypeComboBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="delBtn">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
      <property name="icon">
       <iconset resource="file.qrc">
        <normaloff>:/image/images/del.png</normaloff>:/image/images/del.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="updateBtn">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
      <property name="icon">
       <iconset resource="file.qrc">
        <normaloff>:/image/images/update.png</normaloff>:/image/images/update.png</iconset>
      </property>
     </widget>
     <widget class="QComboBox" name="selDateComBox">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="freshBtn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
      <property name="icon">
       <iconset resource="file.qrc">
        <normaloff>:/image/images/refresh.png</normaloff>:/image/images/refresh.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="managerPage">
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>651</width>
        <height>291</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="insertDataPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>651</width>
         <height>161</height>
        </rect>
       </property>
       <attribute name="label">
        <string>文档信息</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>189</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>资料名称</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dataNameEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>120</y>
          <width>92</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>评定等级</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="dataLeveComBox">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>129</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>资料类型</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="typeIDComBox">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>129</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>课程代码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="courseIDComBox">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>60</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>存储位置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dataURLEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>文件名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fileNameEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="insertTeacherPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>651</width>
         <height>161</height>
        </rect>
       </property>
       <attribute name="label">
        <string>教师信息</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>189</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>教师编号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="teaIdEdit">
           <property name="toolTip">
            <string>输入10位教师编号</string>
           </property>
           <property name="maxLength">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>189</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>教师姓名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="teaNameEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>189</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>联系电话</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="teaNumEdit">
           <property name="maxLength">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>189</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>教师专业</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="teaSpecEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>189</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>教师职称</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="teaTitleEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="insertClassPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>651</width>
         <height>161</height>
        </rect>
       </property>
       <attribute name="label">
        <string>班级信息</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>188</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>班主任编号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="classTeachIDComBox">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>189</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>班级简称</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="classSNameEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>189</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>班级名称</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="classNameEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>189</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>班级编号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="classIdEdit">
           <property name="maxLength">
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="insertCoursePage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>651</width>
         <height>161</height>
        </rect>
       </property>
       <attribute name="label">
        <string>授课信息</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>176</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>教师编号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="courseTeachIDComBox">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>129</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>班级编号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="courseClassIDComBox">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>189</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>课程代码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="courseIdEdit">
           <property name="maxLength">
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>90</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>课时数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="courseHourSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>189</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>考核方式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="courseExaFEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="insertTypePage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>651</width>
         <height>161</height>
        </rect>
       </property>
       <attribute name="label">
        <string>类型信息</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>189</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_23">
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>文档类别</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="typeName"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>189</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_24">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>文档形式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="typeForm"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="manageSureBtn">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QPushButton" name="manageCleanBtn">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>清除</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="insertBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>增加信息</string>
    </property>
    <property name="icon">
     <iconset resource="file.qrc">
      <normaloff>:/image/images/add.png</normaloff>:/image/images/add.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="userQuitLoginBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>未登录</string>
    </property>
    <property name="icon">
     <iconset resource="file.qrc">
      <normaloff>:/image/images/yonghu.png</normaloff>:/image/images/yonghu.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="actionChangePwd"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionChangePwd">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>dataNameEdit</tabstop>
  <tabstop>courseIDComBox</tabstop>
  <tabstop>typeIDComBox</tabstop>
  <tabstop>dataURLEdit</tabstop>
  <tabstop>fileNameEdit</tabstop>
  <tabstop>dataLeveComBox</tabstop>
  <tabstop>manageSureBtn</tabstop>
  <tabstop>insertBtn</tabstop>
  <tabstop>updateBtn</tabstop>
  <tabstop>selectBtn</tabstop>
  <tabstop>selTableComBox</tabstop>
  <tabstop>delBtn</tabstop>
  <tabstop>selTeachComBox</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>selClassComBox</tabstop>
  <tabstop>selTypeComboBox</tabstop>
  <tabstop>manageCleanBtn</tabstop>
 </tabstops>
 <resources>
  <include location="file.qrc"/>
 </resources>
 <connections/>
</ui>
